backend/codesirius/problems/models/reference_solution.py

```
@@ -0,0 +1,22 @@
+"""
+Reference solution model
+"""
+
+from django.db import models
```

Ruff (0.8.2)

`django.db.models` imported but unused

Remove unused import: `django.db.models`

(F401)